ramalama/rag.py

```diff
@@ -85,9 +87,15 @@ def generate(self, args):
         for path in args.PATH:
             self._handle_paths(path)
 
-        ragdb = tempfile.TemporaryDirectory(dir=tmpdir, prefix='RamaLama_rag_')
-        dbdir = os.path.join(ragdb.name, "vectordb")
-        os.mkdir(dbdir)
+        # If user specifies path, then don't use it
+        target_is_oci = self.target[0] != "/"
```

Comment:
Two ideas:

1. I wonder if it's worth supporting explicit identification with an `oci://` vs. `file://` prefix or the like.
2. Support for relative paths

Something like this?

```suggestion
        
        target_is_oci = self.target.startswith("oci://") or (not self.target.startswith("file://") and self.target[0] not in {".", "/"}) 
```

Reply:
Well the first option is covered under the `or` block. But I will add it since it adds some more clarification .